HOMEWORK.js

```diff
@@ -58,31 +58,36 @@ function homework (renderList) {
 	// ◽ super bonus: remove the X if it already has one (put it back to incomplete)
 	function onCompleteTodo (index) {
 		// Add code here
-
+		todos[index].charAt(0) === 'X' ? todos[index] = todos[index].slice(1) : todos[index] = 'X' + todos[index];
 		renderList(todos);
 	}
 
 	// ✅This function should:
 	// ◽ remove from the list all of the completed items (that start with 'X')
 	function onRemoveCompleted () {
 		// Add code here
-
+		todos.filter(item => item.charAt(0) === 'X').map(item => todos.splice(todos.indexOf(item),1));
```

Comment:
This is good, but it is actually possible to do it without the `map` (only the `filter`). Can you find an easier way to use the return value of `filter` to achieve the same effect?

And secondly, if you do need to do the splice, `forEach` might be a more natural array method than `map` because you are _doing_ something and not _returning_ something you would like to put in a new array.